Gruntfile.js

```diff
@@ -104,9 +104,7 @@ module.exports = function(grunt) {
         ignores: [
           'static/js/modules/xpath-element-path.js',
           'tests/karma/q.js',
-          'node_modules/**',
-          'api/node_modules/**',
-          'sentinel/node_modules/**',
+          '**/node_modules/**',
```

Comment:
Probably safe, but perhaps worth bearing in mind that projects like pouchdb keep their own source in a dir called `node_modules`.

Reply:
Yeah. It seems OK now, I guess we can revisit if it causes problems.